libs/auth/src/card.ts

```diff
@@ -3,17 +3,63 @@ import {
   ElectionManagerUser,
   PollWorkerUser,
   SystemAdministratorUser,
-  UserWithCard,
 } from '@votingworks/types';
 
+interface SystemAdministratorCardDetails {
+  jurisdiction: string;
+  user: SystemAdministratorUser;
+}
+
+interface ElectionManagerCardDetails {
+  jurisdiction: string;
+  user: ElectionManagerUser;
+}
+
+interface PollWorkerCardDetails {
+  jurisdiction: string;
+  user: PollWorkerUser;
+  /**
+   * Unlike system administrator and election manager cards, which always have PINs, poll worker
+   * cards by default don't have PINs but can if the relevant system setting is enabled.
+   */
+  hasPin: boolean;
```

Comment:
I consciously decided not to add hasPin to the card details for the other user roles that always have PINs. Didn't want to create the false sense that having a PIN or not was an option for them